src/tests/ftest/mdtest/small.yaml

```diff
@@ -13,13 +13,15 @@ server_config:
       fabric_iface_port: 31317
       log_file: daos_server0.log
       storage: auto
+      log_mask: DEBUG
```

Comment:
do we really want to create huge debug log for this and increase the test time?
users will never enable such debug in their testing, so unless for investigation, we should not merge this IMO.

Reply:
I agree, this PR seems reasonable to gather logs to work on a issue but it doesn't feel like something we should land.

Reply:
I have already repeated the test times, but not reproduce the issue. I will repeat more, but not sure whether it can reproduce the issue or not. If still cannot, we may have to land to wait next failure instance on master.

Reply:
What are the compressed log sizes with this change and one iteration?  i.e. how much data would it make per test-run?

Reply:
Based to current test configuration (2 servers * 2 engines), the compressed logs size for one iteration is about 400MB or less.

Reply:
@brianjmurrell is it OK to add this much logging per test?  Test-tags include pr,daily_regression so this is run often.

Reply:
Do we have any idea how much bigger the resulting, compressed log is as a result?